project1/client/client_util.h

```diff
@@ -0,0 +1,54 @@
+#ifndef PROJECT1_UTIL_H
+#define PROJECT1_UTIL_H
+#include <cstdint>
+#include <string>
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <fstream>
+
+#include "ftp_messages.pb.h"
+#include "../wire_protocol/wire_protocol.h"
+
+namespace client_util {
+inline struct sockaddr_in MakeAddress(uint16_t port) {
```

Comment:
I generally avoid `inline` functions. Usually, the compiler is smart enough to auto-inline code for you anyway when it thinks it can, making this keyword effectively meaningless. To that end, is there any reason you didn't put the definitions for these in a .cpp file?

Reply:
I actually did that because I couldn't get the `client_util.cpp` file to behave properly. I was bound to be doing something wrong.

Reply:
Interesting... What specific issue were you having?